docs/users/cli-usage.md

```diff
@@ -0,0 +1,146 @@
+<h1>Using the ddev command line interface (CLI)</h1>
+
+Type `ddev` in a terminal to see the available ddev commands:
+
+```
+➜  ddev
+This Command Line Interface (CLI) gives you the ability to interact with the ddev to create a local development environment.
+
```

Comment:
It would definitely be useful to automate components of the docs like this if we can determine a reasonable way to do so.